src/ansi-c/CMakeLists.txt

```diff
@@ -3,20 +3,57 @@ generic_flex(ansi_c)
 
 add_executable(converter library/converter.cpp)
 
+set(ansi_c_library_sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/library/cegis.c
```

Comment:
Just curious: is the use of wildcards not possible here?

Reply:
Raw wildcards aren't, because we need to evaluate the list of files at compile time, and wildcards are expanded in a shell at build time. We could use `file(GLOB` like elsewhere, I suppose

Reply:
I think I'd prefer that so as to avoid some maintenance nightmare. All .c files are to be taken into account, so `GLOB` sounds about right.